Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM osrf/ros:humble-desktop-full
+
+# Install dependencies (taken from devcontainer dockerfile)
+RUN DEBIAN_FRONTEND=noninteractive apt-get update -q && \
+    apt-get update -q && \
+    apt-get install -yq --no-install-recommends \
+    lcov \
+    curl \
+    wget \
+    python3-pip \
+    python-is-python3 \
+    python3-argcomplete \
+    python3-colcon-common-extensions \
+    python3-colcon-mixin \
+    python3-rosdep \
+    libpython3-dev \
+    python3-vcstool \
+    ros-humble-tl-expected && \
```

Comment:
@ryan-roche nit: this debian is already listed within `install_spot_ros2.sh`.

Reply:
I'm not seeing these packages within the referenced file. Are they covered by a meta package? If so, are all four lines not needed, then?

Reply:
Gotcha. I think GitHub always shows a minimum of four lines for code references so I misinterpreted it as you also saying to remove things like `python3-rosdep`

Reply:
Addressed. [d3a0c7e](https://github.com/ryan-roche/spot_ros2/commit/d3a0c7e1285e493212e4ad0cc371774bc8c5281d)